projects/ccf-eui/src/app/shared/components/spatial-search-config/spatial-search-config.component.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  HostBinding,
+  Input,
+  Output,
+} from '@angular/core';
+import { MatDialogRef } from '@angular/material/dialog';
+import { ALL_POSSIBLE_ORGANS, OrganInfo } from 'ccf-shared';
+
+
+export type Sex = 'male' | 'female';
+
+@Component({
+  selector: 'ccf-spatial-search-config',
+  templateUrl: './spatial-search-config.component.html',
+  styleUrls: ['./spatial-search-config.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SpatialSearchConfigComponent {
+  @HostBinding('class') readonly className = 'ccf-spatial-search-config';
+
+  @Input() allOrgans: OrganInfo[] = ALL_POSSIBLE_ORGANS;
```

Comment:
Should be named `organs`

Reply:
Actually, you shouldn't be pulling this from ccf-shared. You should get it from the SceneState: <https://github.com/hubmapconsortium/ccf-ui/blob/fe6d65c8cbe3c81e1e99c5986c8ea418da183275/projects/ccf-eui/src/app/core/store/scene/scene.state.ts#L51>.